docs/rfcs/004-issueish-list.md

```diff
@@ -0,0 +1,112 @@
+* Mergability status -> `Able to merge`, links to the [Merging controls at the bottom](https://github.com/atom/github/pull/1503#partial-pull-merging)
+  * "Merge PR" to merge the pull request on GitHub if it is open.
+  * "Close" to close the pull request, unmerged, if it is open.
+  * "Re-open PR" to re-open a pull request if it is closed.
+* `Conversation` with comment count, opens the current PR timeline in a center pane.
+  * Reaction emoji and counts.
+  * Description (PR body) as rendered markdown.
+
+![detail](https://user-images.githubusercontent.com/378023/41140383-368c45d4-6b28-11e8-87c2-d4bc0b47fbe1.png)
+
+## New PR
+
+If no current PR can be found, an "open new pull request" button is shown. If needed it also offers to "Publish" or "Push".
+
+![new pr](https://user-images.githubusercontent.com/378023/41136463-5d8dd3da-6b11-11e8-8e28-72275a691430.png)
+
+When the current branch is the default branch, e.g. `master`, a message is shown that suggests to "Create a new branch".
+
+## Drawbacks
+
+None!
+
+## Rationale and alternatives
+
+Our current GitHub panel focuses on showing you stuff about _the pull request that's associated with your current branch._ The problem is, it's difficult to unambiguously determine that in the general case.
+
+The first thing you see today when you open the GitHub panel on the `master` branch of an active repository is not very helpful:
+
+![wat](https://user-images.githubusercontent.com/17565/40857603-99b92304-65a9-11e8-986e-0f14290bda8a.png)
+
+This is a list of _all pull requests on GitHub that have a head ref called "master", from any head repository_. You can then "pin" any of them to see that pull request's details. This isn't useful on master and it's unclear to users what this is supposed to accomplish. Pinning was intended to be an infrequent edge case when we couldn't find the right PR for a given ref, not the first interaction you have with the package.
+
+Showing a PR list instead provides a uniform, more easily understood entry point to the package's GitHub functionality, and paves the way to other pull-request-focused activities in the future. "All open PRs" seems like a reasonable starting point, and "current PRs" preserves the ability to take advantage of your local editor context.
+
+With that said, the choices for the specific lists we show are a bit arbitrary. We'll need to research and iterate on them quite a bit to find what's most useful for the most people, but for now we need to start with something.
+
+## Unresolved questions
+
+### Before RFC merge:
+
+- [ ] What else from the existing issueish pane should we keep? Comments, timeline events?
+- [ ] Are there other pull request actions it would be useful to support?
+
+### Out of scope:
+
+- [ ] How can we allow a user to customize the lists?
+- [ ] How can we notify a user about updated activity on a visible PR?
+
+## Implementation phases
+
+1. Accordion list infrastructure: search model, collapsible list component.
```

Comment:
are you going to make a generic "list" component that could be re-used elsewhere in the UI?  It seems like we have a lot of "one off" components, and it might make sense to think about what could be made generic.  (Ideally, I'd love a React UI components library for Atom that provides generic modals and lists and forms and other stuff, but one thing at a time.)

Reply:
> Are you going to make a generic "list" component that could be re-used elsewhere in the UI?

Yah, I was mentally planning on having an `AccordionList` component.

> It seems like we have a lot of "one off" components, and it might make sense to think about what could be made generic.

This is true, we don't as of yet have many reusable building block-style components. Does anything come to mind for abstractions we could introduce to consolidate existing ones? That would be a cool way to improve the codebase 🤔 

> Ideally, I'd love a React UI components library for Atom that provides generic modals and lists and forms and other stuff, but one thing at a time.

I've had Secret Plans :tm: to extract our [Atom API integration components](https://github.com/atom/github/tree/master/lib/atom) to their own npm package for a while. We could build on that :smile:

Reply:
👍 I would also love if Atom would provide UI components beyond the basic ones from the [styleguide](https://github.com/atom/styleguide). Then not every package author has to create their own from scratch.

> having an `AccordionList` component

A while (very long time) ago, I made an accordion component: https://github.com/atom/atom-ui/pull/14

![accordion](https://cloud.githubusercontent.com/assets/378023/22718103/da7a8590-ede1-11e6-8a97-ad67b6df9392.gif)

It uses the  `<details>` and `<summary>` HTML elements to provide the collapsing functionality. Not sure if we wanted to use that. Or just initially?

Reply:
yeah, if we care about Atom's "hackability" providing a generic UI toolkit that package authors could leverage seems like a REALLY high value thing.  (And then we can also bake in accessibility / i18n).  But one thing at a time, since creating a UI toolkit is a large undertaking.